CoffeeChat/Controller/GroupsViewController.swift

```diff
@@ -135,55 +138,12 @@ class GroupsViewController: UIViewController {
     }
 
     // MARK: - Search Bar
-    /// Updates the search bar text based on currently selected Groups.
-    private func updateSearchBarText() {
-        if selectedGroups.isEmpty {
-            searchBar.text = ""
-            return
-        }
```

Comment:
This is just because we switched up our search organizations design. So before, it was more complex and Phillip had written code to address that but we don't really need any of that code anymore.